parlai/zoo/model_list.py

```diff
@@ -1491,4 +1491,33 @@
         [labels]: Two young guys with shaggy hair look at their hands while hanging out in the yard.|Two young, White males are outside near many bushes.|Two men in green shirts are standing in a yard.|A man in a blue shirt standing in a garden.|Two friends enjoy time spent together.
         """,
     },
+    {
+        "title": "Gender classifier",
+        "id": "genderation_bias",
+        "path": "zoo:genderation_bias/gender_classifier/model",
+        "agent": "bert_ranker/bi_encoder_ranker",
+        "task": "n/a",
+        "project": 'https://github.com/facebookresearch/ParlAI/tree/master/projects/genderation_bias',
+        "description": "BERT bi-encoder ranker for performing gender classification",
+        "example": "parlai dm --model-file zoo:genderation_bias/gender_classifier/model --task blended_skill_talk --eval-candidates inline --verbose True",
```

Comment:
would it be better to give an example where the classifier is classifying something? rather than ranking BST labels?